<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-03T14:10:18Z" name="Suite" finished-at="2022-08-03T14:11:27Z" duration-ms="68678">
    <groups>
    </groups>
    <test started-at="2022-08-03T14:10:18Z" name="Test2" finished-at="2022-08-03T14:11:26Z" duration-ms="68273">
      <class name="practice.ValidationCheck">
        <test-method is-config="true" signature="initialize()[pri:0, instance:practice.ValidationCheck@1cd201a8]" started-at="2022-08-03T14:10:18Z" name="initialize" finished-at="2022-08-03T14:11:06Z" duration-ms="48235" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="validation()[pri:0, instance:practice.ValidationCheck@1cd201a8]" started-at="2022-08-03T14:11:06Z" name="validation" finished-at="2022-08-03T14:11:13Z" duration-ms="6838" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED COURSESes] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED COURSESes] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at practice.ValidationCheck.validation(ValidationCheck.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practice.ValidationCheck@1cd201a8]" started-at="2022-08-03T14:11:17Z" name="tearDown" finished-at="2022-08-03T14:11:26Z" duration-ms="9152" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- practice.ValidationCheck -->
    </test> <!-- Test2 -->
    <test started-at="2022-08-03T14:10:18Z" name="Test" finished-at="2022-08-03T14:11:27Z" duration-ms="68678">
      <class name="practice.HomePage">
        <test-method is-config="true" signature="initialize()[pri:0, instance:practice.HomePage@6731787b]" started-at="2022-08-03T14:10:18Z" name="initialize" finished-at="2022-08-03T14:11:08Z" duration-ms="49950" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="navigation(java.lang.String, java.lang.String)[pri:0, instance:practice.HomePage@6731787b]" started-at="2022-08-03T14:11:08Z" name="navigation" data-provider="logindetails" finished-at="2022-08-03T14:11:23Z" duration-ms="14521" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Sathish]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[password]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- navigation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practice.HomePage@6731787b]" started-at="2022-08-03T14:11:23Z" name="tearDown" finished-at="2022-08-03T14:11:27Z" duration-ms="4033" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- practice.HomePage -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
